scripts/cc-cli/src/utils/bond.ts

```diff
@@ -0,0 +1,42 @@
+import { ApiPromise } from "creditcoin-js";
+import { initKeyringPair } from "./account";
+
+export async function bond(
+  stashSeed: string,
+  controllerAddress: string,
+  amount: number,
+  rewardDestination: "Staked" | "Stash" | "Controller",
+  api: ApiPromise
+) {
+
+  const amountInMicroUnits = BigInt(amount) * BigInt(1000000000000000000); // Multiply by to convert to micro units
```

Comment:
QE note: need to check what will happen if the amount entered is an invalid number as well as if it is already in a micro-units format (e.g. already has 18 trailing zeroes). 

@pLabarta do you know what are the accepted boundary values here ?

Reply:
Added check for min bond amount (1 CTC), upper bound now checks against account balance